packages/devtools_app/integration_test/test/live_connection/eval_and_browse_test.dart

```diff
@@ -116,5 +124,82 @@ class _EvalTester {
     await tester.pumpAndSettle();
     await simulateKeyDownEvent(LogicalKeyboardKey.enter);
     await simulateKeyUpEvent(LogicalKeyboardKey.enter);
+    await tester.pump(longPumpDuration);
+  }
+
+  Future<void> switchToSnapshotsAndTakeOne() async {
+    // Open memory screen.
+    await switchToScreen(tester, ScreenMetaData.memory);
+
+    // Close warning and chart to get screen.
+    await tapAndPump(find.byKey(DebugModeMemoryMessage.closeKey));
```

Comment:
instead of creating a key, Use flutter test finders to find the button. You can find by icon Icons.close that is a descendent of BannerMessage

Reply:
There are other close icons. For example, to delete snapshot.

Reply:
You can use the descendant finder to ensure that the close icon is a child of BannerMessage. We should avoid creating keys just for testing when a widget can be looked up easily with test finders.